pkgs/tools/misc/sysz/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenvNoCC,  fetchFromGitHub, lib, makeWrapper, bash, gawk, fzf }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "sysz";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "joehillen";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256:0idi94jb075jki1jkm7shmicv77ylm2xjpjq6c868j9w6pvnash0";
+  };
+
+  buildInputs = [ makeWrapper ];
+  preBuild = "patchShebangs README.sh";
+  build = "make all";
```

SYSZ_VERSION is up to date in the `sysz` file, i think the makefile is only meant for the developers

```suggestion

dontBuild = true;
```